api/src/main/java/com/codeforcommunity/rest/subrouter/authenticated/ProtectedSchoolRouter.java

```diff
@@ -175,6 +177,16 @@ private void registerGetBookLog(Router router) {
     getBookFlow.handler(this::handleGetBookLog);
   }
 
+  private void registerGetWithoutLibraryReportAsCsv(Router router) {
+    Route getReportAsCsv = router.get("/reports/without-library/:report_id");
+    getReportAsCsv.handler(this::handleGetWithoutLibraryReportAsCsv);
+  }
+
+  private void registerGetWithLibraryReportAsCsv(Router router) {
+    Route getReportAsCsv = router.get("/reports/with-library/:report_id");
+    getReportAsCsv.handler(this::handleGetWithLibraryReportAsCsv);
+  }
```

Comment:
Should we be returning the CSV by report_id? won't it only have one record? don't we want all reports for a particular school included?

Reply:
I had thought we would only be downloading one at a time? maybe a question for Amanda